salesforcecli/plugin-org
# Sh/improved resume sandbox

### What does this PR do?
Uses the changes from https://github.com/forcedotcom/sfdx-core/pull/944 to warn about multiple sandboxes in a resumable state and assumes the user wants to resume the most recent SandboxProcess.

### What issues does this PR fix or reference?
@W-13635704@

NOTE: This PR should merge after the core PR (944) has been merged and the version of `@salesforce/core` is bumped within this PR.

QA Notes:
1. Link the branch from PR 944 of `@salesforce/core` to this branch and build.
2. If you don't already have a sandbox to use for testing, create one (feel free to use `org resume sandbox` with that sandbox process but the real test is with a refresh).
3. Open the parent org of the sandbox just created (or already created) and go to the sandbox list page.  Refresh the sandbox.
4. Use the `org resume sandbox` command from the build of plugin-org with the sandbox name parameter.  Note the warning about multiple sandboxes in a resumable state.  Run it with and without polling.
5. Use the `org resume sandbox` command from the build of plugin-org with the sandbox process ID.  Note that there is no warning because the ID is unique and identifies the exact sandbox process to report on.
